src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -28,18 +28,19 @@ export class UBAClientWithRefresh extends BaseUBAClient {
   public async update(state: { [chainId: number]: UBAChainState }, forceClientRefresh?: boolean): Promise<void> {
     if (state) {
       await super.update(state);
+      return;
```

Comment:
Does this make sense? When we pass in a state already, then we don't need to continue no?